src/entity/user/UserPost.ts

```diff
@@ -57,4 +59,14 @@ export class UserPost {
     onDelete: 'CASCADE',
   })
   user: Promise<User>;
+
```

Comment:
Shouldn't we save here the amount of the award?

Reply:
it is saved under `UserTransaction`